website/www/site/layouts/partials/feedback.html

```diff
@@ -18,5 +18,5 @@ <h3>{{ T "feedback-title" }}</h3>
     <p class="description">
         {{ T "feedback-description" }}
     </p>
-    <button class="load-button"><a href="mailto:dev@beam.apache.org?subject=Beam Website Feedback">{{ T "feedback-button" }}</a></button>
```

Comment:
This is good. 

Better would be: Is there  a way to embed a very simple form?

Reply:
I will give it try later this week.